delivery.yaml

```diff
@@ -6,14 +6,17 @@ notifications:
 pipeline:
   - id: build
     type: script
-    overlay: ci/python
+    vm_config:
+      type: linux
+      image: cdp-runtime/python-3.9
     commands:
     - desc: "Install dependencies"
-      cmd: pip install -r requirements.txt
+      cmd: |
+        pip install -r requirements.txt
     - desc: "Run Tests"
       cmd: python3 setup.py test
-    - desc: "Check code style"
-      cmd: python3 setup.py flake8
+    # - desc: "Check code style"
+    #   cmd: python3 setup.py flake8
```

Comment:
FYI: I disabled this for now as I don't know how to get it to work.

Reply:
Alternatively, we can replace `Check code style` command with
```yaml
- desc: "Check code style"
  cmd: |
    pip3 install flake8
    flake8
```